lanl/podman-quadlets/inventory/group_vars/ochami/coredhcp.yaml

```diff
@@ -0,0 +1,9 @@
+coredhcp_server_id: '{{ cluster_boot_ip }}'
+coredhcp_dns_server: '{{ cluster_boot_ip }}'
+coredhcp_router: '{{ cluster_boot_ip }}'
+coredhcp_netmask: '255.255.255.0'
+coredhcp_dhcp_pool: '172.16.0.200 172.16.0.250'
+coredhcp_tmp_lease_duration: '5m'
+coredhcp_lease_duration: '1h'
+coredhcp_cache_validity: '30s'
+coredhcp_ipxe_binary: 'ipxe-x86_64.efi'
```

Comment:
Is this needed? The newest CoreDHCP with coresmd will determine this based on the architecture and these binaries are now included in the container as ov v0.1.0.

Reply:
Should be removed now.